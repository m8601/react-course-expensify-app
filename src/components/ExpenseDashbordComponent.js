import React from 'react';
import ExpenseList from './ExpenseList';
import ExpensesListFilters from './ExpensesListFilters';


const ExpenseDashbordComponent = () => (
  <div>
    <ExpensesListFilters/>
    <ExpenseList/>
  </div>
);

export default ExpenseDashbordComponent;
