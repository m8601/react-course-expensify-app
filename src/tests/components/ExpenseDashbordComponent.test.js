import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashbordComponent from '../../components/ExpenseDashbordComponent';


test('should render ExpensesListItem with expense', () => {
  const wrapper = shallow(<ExpenseDashbordComponent />);
  expect(wrapper).toMatchSnapshot();
});