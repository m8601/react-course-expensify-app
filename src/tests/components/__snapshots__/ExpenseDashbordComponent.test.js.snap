// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesListItem with expense 1`] = `
<div>
  <Connect(ExpensesListFilters) />
  <Connect(ExpenseList) />
</div>
`;
